src/histolab/util.py

```diff
@@ -65,7 +72,55 @@ def threshold_to_mask(img: Image.Image, threshold: float) -> np.ndarray:
         if the corresponding input array pixel exceeds the threshold value.
     """
     img_arr = np.array(img)
-    return img_arr > threshold
+    return relate(img_arr, threshold)
+
+
+def polygon_to_mask_array(dims: tuple, vertices: List[tuple]) -> np.ndarray:
+    """
+
+    :param dims:
+    :param vertices:
+    :return:
+    """
+    img = Image.new("L", dims, 0)
+    poly = list(map(tuple, vertices))
+    ImageDraw.Draw(img).polygon(poly, outline=1, fill=1)
+    return np.array(img)
+
+
+CoordinatePair = namedtuple("CoordinatePair", ("x_ul", "y_ul", "x_br", "y_br"))
+
+
+def scale_coordinates(reference_coords, reference_size, target_size):
+    """Compute the coordinates corresponding to a scaled version of the image.
+
+    Parameters
+    ----------
+    reference_coords: tuple, array-like or Coordinates
+        (x, y) pair of coordinates referring to the upper left and lower right corners
+        respectively. The function expects a tuple of four elements or a tuple of two
+        pairs of coordinates as input.
+    reference_size: array_like of int
+        Reference (width, height) size to which input coordinates refer to
+    target_size: array_like of int
+        Target (width, height) size of the resulting scaled image
+
+    Returns
+    -------
+    coords: Coordinates
+        Coordinates in the scaled image
+
+    """
+    assert len(reference_size) == 2
```

Comment:
why assert statements here? I think that considering we have tests we can get rid of this lines

Reply:
ok, so should we write a warning or something?

Reply:
yes let's manage the exceptions as well as you did in the other modules

Reply:
ok, should we write a warning or something?